### PR TITLE
User config

### PR DESCRIPTION
**Pull request by np-ent**
Requesting review

### CODE REVIEW

File: user-service/src/main/java/com/recipe/app/user/User.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "users")
+@RequiredArgsConstructor
```
Comment (CraigSDel) [inline user-service/src/main/java/com/recipe/app/user/User.java:11]: Updated to the builder pattern. e.g.

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder

File: user-service/src/main/resources/application.properties [hunk 2]
```diff
@@ -1 +1,7 @@
-
+server.port=8082
+spring.data.mongodb.authentication-database=admin
+spring.data.mongodb.username=admin
```
Comment (CraigSDel) [inline user-service/src/main/resources/application.properties]: Remove username and password from the application files

File: user-service/src/test/java/com/recipe/app/user/UserMongoRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
+package com.recipe.app.user;
+
```
Comment (CraigSDel) [inline user-service/src/test/java/com/recipe/app/user/UserMongoRepositoryTest.java:2]: remove the extra lines

File: user-service/src/main/java/com/recipe/app/user/User.java (hunk 1 above)
Comment (np-ent) [reply user-service/src/main/java/com/recipe/app/user/User.java:11]: done

**Pull request merged**